jdaviz/configs/default/plugins/viewers.py

```diff
@@ -30,6 +33,25 @@ def __init__(self, *args, **kwargs):
         # Allow each viewer to cycle through colors for each new addition to the viewer:
         self.color_cycler = ColorCycler()
 
+    @property
+    def user_api(self):
+        if isinstance(self, AstrowidgetsImageViewerMixin):
+            expose = ['zoom', 'zoom_level']
+        else:
+            expose = ['set_lims']
```

Comment:
Why `set_lims` and not `set_limits`?

Also, is this unit aware? What if data of native different units are loaded? Or does this only care about the currently *displayed* units?

Would `set_spectral_axis_limits` and `set_flux_limits` make more sense as separate methods for 1D spectrum viewer? Though I am not sure about the 2D spectrum viewer or the Mosviz table viewer.

Reply:
> Why set_lims and not set_limits?

No good reason, that's what we have in the plugin plots, but if we would prefer limits, we can change them both here, I don't think that is considered public API.

> Also, is this unit aware? What if data of native different units are loaded? Or does this only care about the currently displayed units?

This is axes limits (based on the current axes units) - if there is no opposition to this, I'll update the docstring accordingly and throw errors for passing quantities.  We could of course support quantities in the future by converting within the method, I can see how that would be useful.

> Mosviz table viewer.

Good point - I need to either have the table viewer not inherit this method or at least exclude it from the user API.

Reply:
> Why set_lims and not set_limits?

Glue does use `reset_lims` (not limits), so I think that is where the plot options API adopted this.  (Similarly for the choice of `x_min` vs `xmin` to match glue-state).